packages/react/src/primitives/types/image.ts

```diff
@@ -48,6 +54,12 @@ export interface ImageOptions extends ImageStyleProps {
    */
   srcSet?: React.ImgHTMLAttributes<HTMLImageElement>['srcSet'];
 
+  /**
+   * @description
+   * Image will be loaded lazily if set to true.
+   */
+  isLazy?: boolean;
```

Comment:
Not sure if this will scale if we are to support "eager" in the future since having `isEager` will conflict `isLazy` if both are `true`

Reply:
Think we do not need an extra API for `eager` coz these 2 are mutual exclusive and modifies the same `loading` attribute, which by default is `eager` if `undefined`

Reply:
What if additional values are allowed for `loading` in the future?

Reply:
What about just remove this prop then since we already expose the HTML attributes? Users can set the `loading` attribute anyway if necessary

Reply:
Agreed that we just drop this prop since `loading` is already supported natively.

Reply:
Will drop it